lsf/lib/azure.c

```diff
@@ -0,0 +1,131 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <syslog.h>
+#include "azure.h"
+
+#define BUFLEN 1024
+static int status=AZURE_ST_NEEDINIT;
+static int CoreNum=1;
+static char SharedKey[BUFLEN+1]={'k',0};
```

Comment:
I thought a good format should be like:
static char SharedKey[BUFLEN + 1] = { 'k', 0 };